tools/postinstall-android.js

```diff
@@ -65,13 +65,16 @@ const pluginRoot = path.join('plugins', 'com.salesforce');
 const libProjectRoot = path.join('plugins', 'com.salesforce', 'src', 'android', 'libs');
 const appProjectRoot = path.join('platforms', 'android');
 
-console.log('Moving Salesforce libraries to the correct location');
+console.log('Moving Salesforce Mobile SDK for Android Gradle workspace to the correct location');
 shelljs.cp('-R', path.join(libProjectRoot, 'mobile_sdk'), appProjectRoot);
 
 console.log('Fixing root level Gradle file for the generated app');
 replaceTextInFile(path.join(appProjectRoot, 'settings.gradle'), "include \":CordovaLib\"", "");
 
-shelljs.echo("includeBuild(new File(\"mobile_sdk\"))").toEnd(path.join(appProjectRoot, 'settings.gradle'));
+shelljs.echo("def salesforceMobileSdkRoot = new File('mobile_sdk');").toEnd(path.join(appProjectRoot, 'settings.gradle'));
+shelljs.echo("if (salesforceMobileSdkRoot.exists()) {").toEnd(path.join(appProjectRoot, 'settings.gradle'));
+shelljs.echo("  includeBuild(salesforceMobileSdkRoot)").toEnd(path.join(appProjectRoot, 'settings.gradle'));
```

Comment:
Now the `includeBuild` for MSDK composite builds is conditional on the existence of the root directory.